src/System.Web.OData/OData/Builder/Conventions/Attributes/ColumnAttributeEdmPropertyConvention.cs

```diff
@@ -36,13 +38,23 @@ public ColumnAttributeEdmPropertyConvention()
                 return;
             }
 
-            var primitiveProperty = edmProperty as PrimitivePropertyConfiguration;
+            var primitiveProperty = edmProperty as PrimitivePropertyConfiguration;           
```

Comment:
Can you also delete the trailing spaces here?

Reply:
All done.